src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -307,5 +318,35 @@ private static bool SupportsCSharpCodeActions(JToken token)
 
             return providesCodeActions && resolvesCodeActions;
         }
+
+        public override Task<JObject[]> UpdateConfigurationAsync(
+            OmniSharp.Extensions.LanguageServer.Protocol.Models.ConfigurationParams configParams,
+            CancellationToken cancellationToken)
+        {
+            var insertSpaces = _clientOptionsMonitor.InsertSpaces;
+            var tabSize = _clientOptionsMonitor.TabSize;
+
+            var result = new JObject[configParams.Items.Count()];
```

Comment:
Once the method returns `object` this can just be an object list which you add to
```suggestion
            var result = new List<JObject>();
```